### ISSUE TITLE
Touchscreen scrolling stuck in text/image selection mode after slow or stationary finger movement in KDE.

### ISSUE THREAD

**Issue by ZZZXXX0110** [mate-desktop/atril#359]
#### Expected behaviour
When scrolling with a touchscreen, page follows exactly as finger tip moves, no matter how slow the movement is, and no matter how long the finger stops moving but still touching the screen.

#### Actual behaviour
If finger is moving very slowly or being stationary while still touching the screen for a few seconds, touchscreen scrolling no longer works and swiping on the screen triggers text/image selection, until touching the zoom box.

#### Steps to reproduce the behaviour
Scroll a pdf document with a touchscreen very slowly, or hold finger on the screen without moving for a few seconds.

#### MATE general version
KDE Plasma 5.14.5, Qt version 5.11.1.

#### Package version
1.20.2-1

#### Linux Distribution
Kubuntu 18.10

**Comment by ZZZXXX0110** [comment]
The same bug is also present in Evince.

**Comment by ZZZXXX0110** [comment]
Nautilus, being another GTK app, also has this long press for selection feature in its UI. However, Nautilus automatically recovers back to scrolling after finger leaves the screen, yet the same behavior is missing in Atril/Evince.

**Comment by lukefromdc** [comment]
Does anyone here on the team have a touchscreen to test this? @raveit65 ? , @monsta ? I do not have one.

**Comment by ZZZXXX0110** [comment]
I'll be happy to do some testing for you guys with my Surface Pro 6!

**Comment by ZZZXXX0110** [comment]
Alternatively, it is an acceptable work around, to have an option to disable text/image selection, in my opinion.

**Comment by lukefromdc** [comment]
We can certainly use the testing, go for it

**Comment by monsta** [comment]
I don't have a touchscreen, maybe anyone else from @mate-desktop/core-team has?

**Comment by ZZZXXX0110** [comment]
I just noticed that this also seem much more likely to happen, when the long press to select is triggered over an image, as opposed to a piece of text. Or in other words, it seems to only get stuck when long press causes background image to be selected. I noticed that this issue doesn't seem to happen on pdf documents that don't have background image, and touch scrolling also successfully recover when text gets selected, rather than an image.

**Comment by ZZZXXX0110** [comment]
Also, might be related to this: https://gitlab.gnome.org/GNOME/evince/issues/754

**Comment by monsta** [comment]
Well, I'm not familiar with touchscreen or touchpad gestures, but we now have https://github.com/mate-desktop/atril/pull/420 merged, which is the backport of the fix for the mentioned Evince bug. Can you check if it fixes this issue too?

**Comment by JosjaVanBever** [comment]
No, the issue isn't fixed. When hovering too slow over a picture on my dell xps 13 7000 series laptop whit ubuntu 19.10 I'm still stuck in text selection mode, which is rather extremely annoying.

**Comment by funkypitt** [comment]
I can confirm the same is happening with evince and Ubuntu 20.04 after an update. It's extremely frustrating when you have, like I do, many comics in pdf format.
